cms/urls.py

```diff
@@ -157,7 +157,6 @@
     url(r'^api/val/v0/', include('edxval.urls')),
     url(r'^api/tasks/v0/', include('user_tasks.urls')),
     url(r'^accessibility$', contentstore.views.accessibility, name='accessibility'),
-    url(r'^zendesk_proxy/', include('openedx.core.djangoapps.zendesk_proxy.urls')),
```

Comment:
The [failing python tests](https://build.testeng.edx.org/job/edx-platform-python-unittests-pr/48069/) confirm my initial suspicion, the sub-urls defined in [`openedx.core.djangoapps.zendesk_proxy.urls`](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/zendesk_proxy/urls.py) are not being included properly.

Maybe it could be fixed with some tinkering in `PluginURLs.CONFIG`?

Reply:
@efischer19 just to close the loop on this: The issue was a typo in `setup.py` it was not registering the zendesk_proxy app at all. (whoops!) Resolved now.